source/frontend/src/features/properties/map/compensation/CompensationRequisitionYupSchema.ts

```diff
@@ -0,0 +1,18 @@
+import * as yup from 'yup';
+/* eslint-disable no-template-curly-in-string */
+
+export const CompensationRequisitionYupSchema = yup.object().shape({
+  agreementDate: yup.string(),
+  expropriationNoticeServedDateTime: yup.string(),
+  expropriationVestingDateTime: yup.string(),
+  generationDatetTime: yup.string(),
```

Comment:
datettime?

Reply:
will update naming with the next ticket.